pkg/registry/rbac/validation/rule.go

```diff
@@ -408,6 +409,15 @@ func (r *StaticRoles) GetClusterRole(name string) (*rbacv1.ClusterRole, error) {
 	return nil, errors.New("clusterrole not found")
 }
 
+func (r *StaticRoles) GetClusterRoleWithMultiTenancy(tenant, name string) (*rbacv1.ClusterRole, error) {
+	for _, clusterRole := range r.clusterRoles {
+		if clusterRole.Name == name && clusterRole.Tenant == tenant{
+			return clusterRole, nil
+		}
+	}
+	return nil, errors.New("clusterrole not found")
```

Comment:
Minor: better to include the clusterrole name and tenant name in the error message.

Reply:
fixed